tests/test_wave_01.py

```diff
@@ -137,18 +142,21 @@ def test_moves_movie_from_watchlist_to_watched():
     }
 
     # Act
-    updated_data = watch_movie(janes_data, movie_to_watch["title"])
+    updated_data = watched_movie(janes_data, movie_to_watch)
 
     # Assert
     assert len(updated_data["watchlist"]) == 1
     assert len(updated_data["watched"]) == 2
     
-    raise Exception("Test needs to be completed.")
+
+    
+    # raise Exception("Test needs to be completed.")
     # *******************************************************************************************
     # ****** Add assertions here to test that the correct movie was added to "watched" **********
+    assert updated_data["watched"][1] == movie_to_watch
```

Comment:
This works as-is but it would be more "Pythonic" to test using `in`. Then you can check that the `movie_to_watch` is in the updated watchlist as well as the movie that was already there, plus you can verify that the movie is no longer in the watchlist and also that the movie that shouldn't have been moved from the watchlist is still on it:
```python
    assert movie_to_watch in updated_data["watched"]
    assert FANTASY_2 in updated_data["watched"]
    assert FANTASY_1 in updated_data["watchlist"]
    assert movie_to_watch not in updated_data["watchlist"]
```